core/src/packet/packet_ms.cpp

```diff
@@ -432,6 +434,17 @@ AOPacket *PacketMS::validateIcPacket(AOClient &client) const
     return PacketFactory::createPacket("MS", l_args);
 }
 
+QRegularExpressionMatch PacketMS::regexTestimonyJumpCommand(QString message) const
```

Comment:
This function should have a more descriptive name, describing what it does rather than how it is used

Reply:
how about `checkIsTestimonyJumpCommand` ?

Reply:
maybe just `isTestimonyJumpCommand`